src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TermQueryExplainIT.java

```diff
@@ -358,6 +358,18 @@ public void testNestedSingleOrderBy() throws IOException {
         assertThat(result, containsString("\"nested\":{\"path\":\"projects\""));
     }
 
+    @Test
+    public void testNestedIsNotNull() throws IOException {
+        String explain = explainQuery(
+                "SELECT e.name " +
```

Comment:
Should the "SELECT" portion be inlined with the rest of the query?

Reply:
Done.